cdap-app-fabric/src/main/java/co/cask/cdap/config/DefaultConfigStore.java

```diff
@@ -80,7 +81,10 @@ public ConfigTable get() {
   }
 
   public static void setupDatasets(DatasetFramework dsFramework) throws DatasetManagementException, IOException {
-    dsFramework.addInstance(Table.class.getName(), configStoreDatasetInstanceId, DatasetProperties.EMPTY);
+    dsFramework.addInstance(Table.class.getName(), Id.DatasetInstance.from(
```

Comment:
if this is all we're doing, do we need this method at all?  Nothing here is specific to DefaultConfigStore right?